nautobot/project-static/css/base.css

```diff
@@ -664,3 +573,103 @@ td span.hover_copy:hover .hover_copy_button {
     max-height: calc(100vh - 212px);
     overflow-y: scroll;
 }
+
+/* Navbar */
+.navbar-fixed-left {
+    border: none;
+    border-radius: 0;
+    bottom: 0;
+    left: 0;
+    margin-bottom: 0;
+    margin-right: 20px;
+    overflow: auto;
+    position: fixed;
+    top: 0;
+    width: 220px;
+    z-index: 1030; /* `1030` is taken from Bootstrap's `navbar-fixed-top` */
+}
+
+.navbar-fixed-left > .container-fluid {
+    padding-left: 0;
+    padding-right: 0;
+}
+
+.navbar-fixed-left .navbar-header {
+    float: none;
+    margin: 0;
+}
+
+.navbar-fixed-left .navbar-header > .navbar-brand {
+    height: auto;
+    margin: 0;
+    padding: 29px 60px 10px 20px;
+}
+
+.navbar-fixed-left .navbar-form {
+    border: none;
+    box-shadow: none;
+    margin: 0;
+    padding: 10px 20px 20px 20px;
+}
+
+.navbar-fixed-left .navbar-form input,
+.navbar-fixed-left .navbar-form button {
+    padding: 7px 10px;
+}
+
+.navbar-fixed-left .navbar-nav {
+    margin: 0;
+    width: 100%;
+}
+
+.navbar-fixed-left .navbar-nav > li {
+    width: 100%;
+}
+
+.navbar-fixed-left .navbar-nav:last-of-type > .dropdown > .dropdown-toggle {
+    text-transform: uppercase;
+}
+
+.navbar-inverse .navbar-nav > li > a {
+    color: #999999;
+    transition: color 200ms ease, background-color 200ms ease;
+}
+
+.navbar-inverse .navbar-nav > li > a:focus,
+.navbar-inverse .navbar-nav > li > a:hover {
+    background-color: #151e2a;
+}
+
+.navbar-fixed-left .navbar-nav > li > a > i.mdi {
+    margin-right: 7px;
+}
+
+.navbar-fixed-left .navbar-nav > .dropdown > .dropdown-toggle > img {
+    margin-right: 10px;
+}
+
+.navbar-inverse .navbar-nav > .dropdown > .dropdown-menu {
+    background-color: #151e2a;
+    border: none;
+    min-width: auto;
+    width: 100%;
+}
+
+.navbar-inverse .navbar-nav > .dropdown > .dropdown-menu .divider {
+    background-color: #505d68;
+}
+
+.navbar-inverse .navbar-nav > .dropdown > .dropdown-menu > li > a {
+    margin-right: 0;
+    transition: background-color 200ms ease;
+    white-space: normal;
+}
```

Comment:
Thoughts on maybe adding `text-indent: -10px; margin-left: 10px` here, or similar, to give a hanging indent when menu items wrap?

![image](https://github.com/nautobot/nautobot/assets/5603551/fe5fc308-8ea4-43af-a5a5-93ec6b00b50a)

Reply:
I see your point, though to be honest I'm not entirely sold on the text indent. How about some slight line height tweaks instead?

![image](https://github.com/nautobot/nautobot/assets/117445994/1fb43aa4-5970-46ea-a1fc-4d6c1a56fd7e)

Reply:
I like that :)

Reply:
Ok, implemented :+1: